compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeLayer.desktop.kt

```diff
@@ -111,6 +112,17 @@ internal class ComposeLayer {
             if (a11yDisabled) return null
             val controller =
                 scene.mainOwner?.accessibilityController as? AccessibilityControllerImpl
+            controller?.parent = component.parent
+            val resetFocusTimer = Timer(100) {
```

Comment:
If I am not mistaken, this will be fired on every focus change, even if we don't use accessebility?

If this is true, the program became unresponsive for 100ms on every focus change

Reply:
Where timeout come from?